trento/xml/article_sap_trento.xml

```diff
@@ -1590,7 +1590,7 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
           </step>
           <step>
             <para>Execute the script:</para>
-            <screen>&prompt.root;<command>./trento-support.sh</command> --output file.tgz --collect all</screen>
+            <screen>&prompt.root;<command>./trento-support.sh</command> --output file-tgz --collect all</screen>
```

Comment:
I propose to also add a mention to 
```
KUBECONFIG=/etc/rancher/k3s/k3s.yaml
```
as the script internally is using tools that relay on that

Reply:
@mpagot I've added this in commit db7d7f0. Ok?